staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -29,11 +29,19 @@ const (
 
 	// owner: @tallclair
 	// alpha: v1.5
+	// beta: v1.6
 	//
 	// StreamingProxyRedirects controls whether the apiserver should intercept (and follow)
 	// redirects from the backend (Kubelet) for streaming requests (exec/attach/port-forward).
 	StreamingProxyRedirects utilfeature.Feature = "StreamingProxyRedirects"
 
+	// owner: @tallclair
+	// alpha: v1.9
+	//
+	// ValidateProxyRedirects controls whether the apiserver should validate that redirects are only
+	// followed to the same host. Only used if StreamingProxyRedirects is enabled.
```

Comment:
what's the plan for StreamingProxyRedirects? is that going away or stuck in beta?

Reply:
I'd like it to go away eventually, but version skew is a problem since it's required for older kubelets (I think anything older than 1.11)

Reply:
Last time when we discussed this, the conclusion was that once authentication is resolved, we can switch back to redirect. https://github.com/kubernetes/kubernetes/pull/64006#issue-188901933
Is that not the case now?

Reply:
Authentication could fix this issue, if we required the certificate to be signed / associated with the node that was originally contacted. I'll follow up on high that might be done.